src/routes/Repositories/Repositories.tsx

```diff
@@ -221,7 +216,7 @@ export default function Repositories(props: RepositoryListProps) {
                     name: repo.name,
                     namespace: repo.namespace,
                     path: repo.namespace + '/' + repo.name,
-                    visibility: repo.is_public ? 'public' : 'private',
```

Comment:
`visibility` should be displayed as a `string` instead of boolean as this variable is used in `<Tbody>` further down to be displayed in the UI. Could you address this?